server/paymentProviders/stripe/webhook.ts

```diff
@@ -31,6 +31,22 @@ import * as virtualcard from './virtual-cards';
 
 const debug = debugLib('stripe');
 
+const coercePaymentMethodType = (paymentMethodType: Stripe.PaymentMethod.Type): PAYMENT_METHOD_TYPE => {
+  switch (paymentMethodType) {
+    case 'card':
+      return PAYMENT_METHOD_TYPE.CREDITCARD;
+    case 'sepa_debit':
+      return PAYMENT_METHOD_TYPE.SEPA_DEBIT;
+    case 'bacs_debit':
+      return PAYMENT_METHOD_TYPE.BACS_DEBIT;
+    case 'us_bank_account':
+      return PAYMENT_METHOD_TYPE.US_BANK_ACCOUNT;
+    case 'alipay':
+      return PAYMENT_METHOD_TYPE.ALIPAY;
+    default:
+      return null;
```

Comment:
What would be the consequence of setting the type to null?
With the exception of card/creditcard, we are using the same names as Stripe and might continue to do so for new types we implement right?

Reply:
Maybe it's sensible to have that as a default.

```
default:
   return paymentMethodType
```

Reply:
And add a warning if we're missing the matching constant on our side?